gengokit/httptransport/templates.go

```diff
@@ -3,102 +3,102 @@ package httptransport
 // ServerDecodeTemplate is the template for generating the server-side decoding
 // function for a particular Binding.
 var ServerDecodeTemplate = `
-{{ with $binding := .}}
-// DecodeHTTP{{$binding.Label}}Request is a transport/http.DecodeRequestFunc that
-// decodes a JSON-encoded {{ToLower $binding.Parent.Name}} request from the HTTP request
-// body. Primarily useful in a server.
-func DecodeHTTP{{$binding.Label}}Request(_ context.Context, r *http.Request) (interface{}, error) {
-	var req pb.{{GoName $binding.Parent.RequestType}}
-	err := json.NewDecoder(r.Body).Decode(&req)
-	// err = io.EOF if r.Body was empty
-	if err != nil && err != io.EOF {
-		return nil, errors.Wrap(err, "decoding body of http request")
-	}
-
-	pathParams, err := PathParams(r.URL.Path, "{{$binding.PathTemplate}}")
-	_ = pathParams
-	if err != nil {
-		fmt.Printf("Error while reading path params: %v\n", err)
-		return nil, errors.Wrap(err, "couldn't unmarshal path parameters")
-	}
-	queryParams, err := QueryParams(r.URL.Query())
-	_ = queryParams
-	if err != nil {
-		fmt.Printf("Error while reading query params: %v\n", err)
-		return nil, errors.Wrapf(err, "Error while reading query params: %v", r.URL.Query())
+{{- with $binding := . -}}
+	// DecodeHTTP{{$binding.Label}}Request is a transport/http.DecodeRequestFunc that
+	// decodes a JSON-encoded {{ToLower $binding.Parent.Name}} request from the HTTP request
+	// body. Primarily useful in a server.
+	func DecodeHTTP{{$binding.Label}}Request(_ context.Context, r *http.Request) (interface{}, error) {
+		var req pb.{{GoName $binding.Parent.RequestType}}
+		err := json.NewDecoder(r.Body).Decode(&req)
+		// err = io.EOF if r.Body was empty
+		if err != nil && err != io.EOF {
```

Comment:
Why have these templates all been indented? Aren't they going to be unindented in the actual code?